_maps/kilostation.json

```diff
@@ -0,0 +1,25 @@
+{
+	"version": 1,
+	"map_name": "Kilo Station",
+	"map_path": "custom",
```

Comment:
this wont work
it needs to be the folder path to the map starting from where the .json is (excluding the file name)